### ISSUE TITLE
[Bug] Missing Add method for AbsoluteLayout that takes 3 args

### ISSUE THREAD

**Issue by davidortinau** [dotnet/maui#3094]
### Description

<img width="834" alt="Screen Shot 2021-10-23 at 9 40 58 AM" src="https://user-images.githubusercontent.com/41873/138561013-8ace6d81-96ff-4843-abd5-ee75ae836537.png">

```
using System;
using Microsoft.Maui; 
using Microsoft.Maui.Controls;
using Microsoft.Maui.Graphics;
using Microsoft.Maui.Layouts;

namespace ControlGallery.Pages.Layouts.AbsoluteLayout
{
    public partial class ProportionalCoordinateCalcDemoPage : ContentPage
    {
        public ProportionalCoordinateCalcDemoPage()
        {
            InitializeComponent();

            Rectangle[] fractionalRects =
            {
                new Rectangle(0.05, 0.1, 0.90, 0.1),    // outer top
                new Rectangle(0.05, 0.8, 0.90, 0.1),    // outer bottom
                new Rectangle(0.05, 0.1, 0.05, 0.8),    // outer left
                new Rectangle(0.90, 0.1, 0.05, 0.8),    // outer right

                new Rectangle(0.15, 0.3, 0.70, 0.1),    // inner top
                new Rectangle(0.15, 0.6, 0.70, 0.1),    // inner bottom
                new Rectangle(0.15, 0.3, 0.05, 0.4),    // inner left
                new Rectangle(0.80, 0.3, 0.05, 0.4),    // inner right
            };

            foreach(Rectangle fractionalRect in fractionalRects)
            {
                Rectangle layoutBounds = new Rectangle
                {
                    // Proportional coordinate calculations
                    X = fractionalRect.X / (1 - fractionalRect.Width),
                    Y = fractionalRect.Y / (1 - fractionalRect.Height),
                    Width = fractionalRect.Width,
                    Height = fractionalRect.Height
                };

                absoluteLayout.Children.Add(new BoxView
                {
                    Color = Colors.DarkBlue
                }, layoutBounds, AbsoluteLayoutFlags.All);
            }
        }

        void OnContentViewSizeChanged(object sender, EventArgs args)
        {
            ContentView contentView = (ContentView)sender;

            // Figure has an aspect ratio of 2:1
            double height = Math.Min(contentView.Width / 2, contentView.Height);
            absoluteLayout.WidthRequest = 2 * height;
            absoluteLayout.HeightRequest = height;
        }
    }
}
```

### Expected Behavior

this works in XF so ought to be available in maui as well

### Actual Behavior

error bc it's missing

### Basic Information

- Version with issue: pre10.1957

**Comment by shimingsg** [comment]
Verified Repro with windows. Repro project available in Maui Triage Teams channel:
https://microsoft.sharepoint.com/:u:/t/Mauitriage/EZVl2ML5JNNKim_MWY4DD5kBTcrmLwBnJCTxV0IvL3gr9A?e=v16U2W

**Comment by jfversluis** [comment]
Fixed by #13408
